manifest_staging/charts/gatekeeper/templates/gatekeeper-mutating-webhook-configuration-mutatingwebhookconfiguration.yaml

```diff
@@ -15,10 +15,14 @@ webhooks:
   - v1
   - v1beta1
   clientConfig:
+    {{- if .Values.webhookURL }}
```

Comment:
these need to be regenerated. can you run `make manifests` please?

Reply:
oops forgot about that. Ran and pushed changes